src/translations/fr/dashboard.json

```diff
@@ -13,6 +13,7 @@
   "latestversionscommits.buttonblock.label": "Explorer",
   "latestversionscommits.buttonblock.link.title": "Explorer les versions de documents",
   "latestversionscommits.buttonblock.title": "Nous avons enregistré {{count}} versions de documents",
+  "latestversionscommits.buttonblock.title_plural": "__STRING_NOT_TRANSLATED__",
```

Comment:
Why is this added in this PR? It should in any case be translated.

Reply:
I think it's because the automatic generation of the key for the plural was introduced after writing the key for the singular. And the key was generated without me seeing it when I ran the command  `npm run i18n`.